src/EPR.RegulatorService.Facade.API/Controllers/OrganisationsSearchController.cs

```diff
@@ -225,9 +239,12 @@ public async Task<IActionResult> AddRemoveApprovedUser(FacadeAddRemoveApprovedPe
             };
             
             _messagingService.SendEmailToInvitedNewApprovedPerson(emailModel);
-            _logger.LogInformation($@"Email sent to Invited new approved person. 
+
+            string logData = $@"Email sent to Invited new approved person. 
                                    Organisation external Id: {request.OrganisationId}
-                                   User: {request.InvitedPersonFirstName} {request.InvitedPersonLastName}");
```

Comment:
The standard way to use Microsoft logger in other project is as follows
_logger.LogInformation("Email sent to Invited new approved person. Organisation external Id: {OrganisationId}, User: {InvitedPersonFirstName} {InvitedPersonLastName}",
 request.OrganisationId, request.InvitedPersonFirstName, request.InvitedPersonLastName);